.github/workflows/lsp-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Language server tests
+
+on:
+  workflow_call:
+
+jobs:
+  run:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Uninstall packages Ubuntu
```

Comment:
They are needed because the language server is able to use multiple different compilers for syntax checking. This is supposed to test that it correctly detects which tools are absent or present, and responds appropriately

Reply:
Perhaps a comment in the yaml file would be useful, because it does looks a bit strange.

Reply:
Okay, the comment is added!